### PR TITLE
Remove slide deck reference from design.login.gov.

### PR DESCRIPTION
**Pull request by juliaelman**
Removing the reference to an outdated slide deck from design.login.gov - new year, new deck! 🌮

### CODE REVIEW

**Review by nickttng — APPROVED** [review]
✂️ ✂️ ✂️ 👍🏼

File: docs/brand.md [hunk 1]
```diff
@@ -132,7 +132,3 @@ Illustration is most effective when it is precise in choosing a single message t
   <img src="{{ site.baseurl }}/assets/img/illustrations/fail.svg" role="img" alt="Failure illustration" class="display-block grid-col flex-auto flex-align-center">
   <img src="{{ site.baseurl }}/assets/img/illustrations/id-confirm.svg" role="img" alt="Confirmed identification illustration" class="display-block grid-col flex-auto flex-align-center">
 </div>
-
-## Slide decks
```
Comment (aduth) [inline docs/brand.md:136]: Per failing build: Need to remove the `sidenav` entry for Slide decks from frontmatter at the top of the file.

Comment (juliaelman) [reply docs/brand.md:136]: Fixed here 86e76fb

**Review by aduth — APPROVED** [review]
Visual regression is expected since we're changing page content.

**Pull request merged**